src/app/core/navigation.service.ts

```diff
@@ -17,6 +18,15 @@ export class NavigationService {
     this.goToUrl('/topics/new/collection');
   }
 
+
+  public goToTopic(topic: string | Topic): void {
+    if (topic instanceof Topic) {
+      this.goToUrl(`/topics/details/${topic.title}`);
```

Comment:
I just realised that we probably should urlencode topic titles, as they might contain characters that are not allowed (e.g. `[`)

Reply:
agreed & fixed